### ISSUE TITLE
Action to move forwards and backwards through Windows Terminal instances

### ISSUE THREAD

**Issue by bavis-m** [microsoft/terminal#7848]
It would be nice to have a bind-able action that would select the next (or previous) running Windows Terminal instance (window). I often find when I'm editing a lot of code I have a terminal window with Vim on each of my monitors, which I have to Alt-Tab back and forth between. However, sometimes I will have opened some documentation or other various windows since the last time I visited a particular WT instance, and I have to expend a lot more mental energy figuring out exactly how many Alt-Tabs to get back to my 2nd terminal window. I used this all the time when I used mintty, which exposed it under the Ctrl-Tab key (which is already default bound to switching tabs in WT). Maybe Ctrl-Alt-Tab or something similar?

# Proposed technical implementation details (optional)

**Comment by zadjii-msft** [comment]
This isn't the silliest idea, once #5000 lands. We'll need that for coordinating this action across processes.

Though, one question - wouldn't it be easier to just have one Terminal window with lots of tabs in it, and then use the already existing `nextTab`/`prevTab` actions?

**Comment by bavis-m** [comment]
I use tabs for in my terminal on a single window (editor in one tab, logs
in another, command line in a third, for example), but if I want terminals
in multiple monitors it is a pain to get a single terminal window to fit
(especially across different dimension/DPI displays)

**Comment by zadjii-msft** [comment]
> if I want terminals in multiple monitors it is a pain to get a single terminal window to fit (especially across different dimension/DPI displays)

Okay that makes perfect sense. Thanks for following up!
